src/ISA/ISA.Json/ARCtrl.ISA.Json.fsproj

```diff
@@ -40,8 +40,9 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="NJsonSchema" Version="10.8.0" />
-    <PackageReference Include="Thoth.Json" Version="10.1.0" />
-    <PackageReference Include="Thoth.Json.Net" Version="11.0.0" />
+    <PackageReference Include="Thoth.Json.Core" Version="0.2.1" />
+    <PackageReference Include="Thoth.Json.JavaScript" Version="0.1.0" />
```

Comment:
No python package? I have not looked into this, just wondering

Reply:
This is only the thoth update, without any python integration. The will come in the next, separate PR.

Reply:
Ohhhh!